js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -195,6 +192,22 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating shareInfo function to MiniAppBridge object.
+   * This function does not return anything back on success.
+   * @param {info} The shared info object.
+   */
+  shareInfo(info: ShareInfoType) {
+    return new Promise<undefined>((resolve, reject) => {
+      return this.executor.exec(
+        'shareInfo',
+        info,
```

Comment:
Can we change this `info` to `shareInfo` or anything you feel that is a little more descriptive? Because I guess iOS works a little differently than android in this case. We need to add this variable in **[RequestParameters](https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/Models/MiniAppJavaScriptMessageInfo.swift#L10)**

Reply:
Yeah let me change it.